package.json

```diff
@@ -23,35 +23,36 @@
     "lint:fix": "prettier --check \"**/*.ts\" \"**/*.md\" \"!**/lib/**\" && tslint --project ./tsconfig.json --fix",
     "publish:unstable": "lerna publish --canary --dist-tag=unstable --preid=unstable --yes --exact",
     "publish:release": "lerna publish from-git --yes",
-    "version:release": "lerna version minor --yes --exact --no-push"
+    "version:release": "lerna version minor --yes --exact --no-push",
+    "prepare": "husky install"
   },
   "devDependencies": {
-    "@commitlint/cli": "8.1.0",
-    "@commitlint/config-conventional": "8.1.0",
-    "cz-conventional-changelog": "3.0.2",
-    "husky": "3.0.4",
-    "jest": "24.9.0",
-    "jest-html-reporters": "1.2.0",
-    "lerna": "3.16.4",
-    "prettier": "1.19.1",
-    "pretty-quick": "1.11.1",
-    "ts-jest": "24.0.2",
-    "tslint": "5.16.0",
+    "@commitlint/cli": "17.0.2",
+    "@commitlint/config-conventional": "17.0.2",
+    "cz-conventional-changelog": "3.3.0",
+    "husky": "8.0.1",
+    "jest": "28.1.1",
+    "jest-html-reporters": "3.0.9",
+    "lerna": "5.1.2",
+    "prettier": "2.6.2",
+    "pretty-quick": "3.1.3",
+    "ts-jest": "28.0.5",
+    "tslint": "5.20.1",
     "tslint-config-prettier": "1.18.0",
-    "tslint-consistent-codestyle": "1.15.1",
-    "tslint-immutable": "6.0.0",
-    "typescript": "3.7.5"
+    "tslint-consistent-codestyle": "1.16.0",
+    "tslint-immutable": "6.0.1",
+    "typescript": "4.3.5"
   },
-  "husky": {
-    "hooks": {
-      "commit-msg": "[[ -n $HUSKY_BYPASS ]] || commitlint -E HUSKY_GIT_PARAMS",
-      "pre-commit": "[[ -n $HUSKY_BYPASS ]] || pretty-quick --staged"
-    }
-  },
-  "commitlint": {
-    "extends": [
-      "@commitlint/config-conventional"
-    ]
+  "@comment: resolutions": [
+    "packages listed below resolve high/critical vulnerabilities"
+  ],
+  "resolutions": {
+    "@commitlint/cli/**/ini": "1.3.6",
```

Comment:
This is neat, you can target the resolution so isn't global. Awesome

Reply:
Credit to @Marcus-Mattr  learned it from him on his PR